packages/sui-test/Dockerfile

```diff
@@ -2,8 +2,6 @@ FROM cypress/base
 
 WORKDIR /usr/src
 
-ARG CYPRESS_VERSION=3.8.3
-
-RUN npm install --force-only @s-ui/test cypress@$CYPRESS_VERSION
+RUN npm install --force-only @s-ui/test
```

Comment:
Why are you remove the installation of Cypress ?!

Reply:
didn't catch at first that this is to avoid lazy load the package, isn't it? gonna roll it back

Reply:
If we remove cypress, where will it be then? In the base image of docker there is no cypress.

In the [documentation](https://hub.docker.com/r/cypress/base/) it recommends using this:

`FROM cypress/base:10
RUN npm install --save-dev cypress
RUN $(npm bin)/cypress verify
RUN $(npm bin)/cypress run`

Reply:
it's being lazy loaded by the sui-test itself https://github.com/SUI-Components/sui/blob/master/packages/sui-test/bin/sui-test-e2e.js#L89

Reply:
ok!!!

Reply:
🎉